frontend/awx/views/schedules/wizard/ScheduleEditWizard.cy.tsx

```diff
@@ -170,7 +170,7 @@ describe('ScheduleAddWizard', () => {
       });
 
       cy.get('[data-cy="name"]').type('Test Schedule');
-      cy.selectDropdownOptionByResourceName('timezone', 'Zulu');
+      cy.selectSingleSelectOption('timezone', 'Zulu');
       cy.clickButton(/^Next$/);
```

Comment:
should this pass in `'[data-cy="timezone"]'` instead of 'timezone'?

Reply:
you're right! thanks